user-access-commons/src/test/java/no/unit/nva/database/DatabaseServiceImplTest.java

```diff
@@ -74,56 +85,45 @@ public void getRoleLogsWarningWhenNotFoundExceptionIsThrown() throws InvalidEntr
             StringContains.containsString(DatabaseServiceImpl.ROLE_NOT_FOUND_MESSAGE));
     }
 
-    @Test
-    public void getUserLogsWarningWhenNotFoundExceptionIsThrown() {
```

Comment:
logging is the responsibility of the Handler-template, so we remove the logging and the the test from here, to avoid double logging of the same event.